app/controllers/users.js

```diff
@@ -1,19 +1,17 @@
 const userService = require('../services/users');
-const { databaseError } = require('../errors');
-const errorMsg = require('../constants/errorMessages');
 const varParse = require('../serializers/varParse');
 
-exports.signUp = async (req, res) => {
+exports.signUp = async (req, res, next) => {
   try {
-    const userCreated = await varParse.nameParse({
+    const userCreated = varParse.nameParse({
       firstName: req.body.first_name,
       lastName: req.body.last_name,
       email: req.body.email,
       password: req.body.password
     });
```

Comment:
just pass to the `nameParse` all the `req.body`

Reply:
This should be like this? 
`firstName: req.body`